src/Uno.UI/UI/Xaml/StateTriggerBase.cs

```diff
@@ -22,9 +22,9 @@ protected StateTriggerBase()
 			);
 		}
 
-		protected void SetActive(bool active)
+		protected void SetActive(bool IsActive)
```

Comment:
This is some very curious casing. Is it this way it's named in winui?

Reply:
Unfortunately, yes:

https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.statetriggerbase.setactive?view=windows-app-sdk-1.3